util/get-net.js

```diff
@@ -0,0 +1,20 @@
+var get = require('lodash.get')
+
+module.exports = function getNet (config) {
+  var connection = get(config, 'connections.incoming.net', [])
+    .find(function (transport) {
+      return isAccessible(transport)
+    })
+
+  if (!connection) return {}
+
+  return {
+    host: connection.host,
+    port: connection.port
+  }
+}
+
+function isAccessible (transport) {
+  return transport.scope === 'private' || transport.scope.includes('private') ||
+    transport.scope === 'public' || transport.scope.includes('public')
+}
```

Comment:
I'm not 100% sure how to choose which transportSetting within `connections.incoming.net` array we should be using for `config.host` and `config.port`

Here's I'm getting the first transportSetting which is .. accessible to the outside, which I think means a public or private scope.

Reply:
you didn't answer this question .. I hope it's fine @dominictarr